resources/public/api/conf/2.0/application.yaml

```diff
@@ -1147,7 +1153,7 @@ paths:
             example: 2023-06-15T09:00:00Z
       responses:
         '200':
-          description: A successful get all messages related to arrival response
+          description: A successful get includes the total count of the number of messages and all messages related to arrival response
           content:

```

Comment:
Same issue as above.

Suggestion 1:
A successful get all messages related to arrival response

Suggestion 2:
A successful get all messages related to arrival response, which may include the total count of the number of messages

Reply:
Original - :+1: